src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -9,6 +9,8 @@
 using Xunit;
 using Microsoft.Xunit.Performance;
 
+#pragma warning disable 0618 // String.Copy is now obsolete
```

Comment:
Why do we need perf tests for Copy?

Reply:
I think the tests are for other `string` APIs that have fast-paths when the two instances being compared are identical, and we wanted in our benchmarks to make sure this fast-path wasn't hit.

@adamsitnik - Do you have context on these?

Reply:
The old benchmarks were using `string.Copy` only to prepare the test data, the method itself did not have benchmarks. It's also not benchmarked in the performance repo https://github.com/dotnet/performance/blob/master/src/benchmarks/micro/corefx/System.Runtime/Perf.String.cs